creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1202,8 +1228,13 @@ class SnapbackSM {
       // Log decision tree
       this._printStateMachineQueueDecisionTree(decisionTree, jobId)
 
+      // The next job should start processing where this one ended or loop back around to the first user
+      const lastProcessedUser = nodeUsers.pop() || { user_id: 0 }
```

Comment:
ah ok looks like we're not using the modulo logic here.. anyway, seems like the default option of starting at id = 0 might cause an artificial "bias processing of early users". should this default also be randomized?

Reply:
synced offline about how this is just for restarting at user_id=0 after cycling through the pages of all other users